CIP-????/README.md

```diff
@@ -0,0 +1,530 @@
+must also specify the value of the UTxO that will eventually go there; these placeholders are
+necessary to finish balancing the intent transaction. But this creates problems for potential
+counter-parties: 
+
+- Only UTxOs that exactly fit those placeholders can be used
+- The counter-party can't add or remove placeholders after the fact
+
+Imagine if Alice wants to swap 100 ADA for 50 DJED; this says nothing about how the counter-party
+will satisfy her swap. Perhaps Bob (the counter-party) wants to provide the DJED, but it is
+currently broken up across three UTxOs. And perhaps he wants the 100 ADA he receives to be stored in
+two separate outputs. If Alice specifies only one input placeholder and one output placeholder in
+her intent transaction, Bob might be unable to act as her counter-party. If Bob really wanted to
+still trade with Alice, he would have to find a way to merge his inputs and outputs into one input
+and one output. This may not always be possible and Bob may have to give up on being Alice's
+counter-party. In other words, because Alice was forced to specify placeholders in her intent, she
+missed out on a potential trading partner.
+
+DApps would also be unable to be direct liquidity sources for these intents due to not always being
+able to match the value specified in each placeholder. If the input place holder is for 10.3 ADA but
+the liquidity source UTxO actually has 20.5 ADA, this liquidity source cannot be used despite it
+having enough liquidity. The Validation Zones CIP argues that DApps can just add support for
+breaking-off/merging UTxOs so that the liquidity source UTxOs can be forced into the proper forms
+for the required placeholders. However, this would dramatically increase the complexity and attack
+surface of DApps that do this. And this added risk to users is just to compensate for a limitation
+in the Validation Zones' design.
+
+In addition to the counter-party issue, there must be a way for the ledger to actually support and
+satisfy these placeholders. To accomplish this, the Validation Zones CIP introduces zones, unlocked
+requests, and unlocked outputs. These new features require adding a lot of extra complexity to the
+ledger and makes using Validation Zones intents a very complicated experience. For example,
+satisfying a single intent actually requires multiple transactions (packaged as a zone).
+
+Again, the counter-party issue and extra complexity in Validation Zones are simply due to not being
+able to accurately model unbalanced intents. Since this CIP *can* model unbalanced intents, it can
+cover all of the same use cases while only adding a fraction of the complexity.
+
+It is also worth noting that, since the Validation Zones design uses otherwise unnecessary
+placeholders and extra transactions just to satisfy these placeholders, the impact Validation Zones
+would have on the average block size is likely equivalent to, if not larger than, the impact this
+CIP would introduce.
+
```

Comment:
Compared to the validation zones approach, this approach appears to be somewhat simpler indeed, but it also has some restrictions. For example, this approach doesn't allow for atomic batch transactions in general. A transaction piece cannot spend an output generated by another transaction piece in the same transaction. I see this as a major drawback, and it should be discussed here.

Reply:
Atomic batch transactions seems orthogonal to intents (it isn't mentioned in the CPS), but I can still discuss it in the CIP.

@polinavino Perhaps there is a way to use transaction pieces as the foundation for zones?